### PR TITLE
fix: Use lockfile as is in CI

### PR DESCRIPTION
**Pull request by riodeuno**
## Description

Update github workflows to use the lockfile in the codebase and not update it.

`yarn install` can potentially override the specifications in the `yarn.lock` file. This is not ideal for CI or for creating release builds. Release builds were always using the `--frozen-lockfile` option, while the CI wasn't. This PR adds this option to the CI's `yarn install` commands.

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- If the CI works properly with these updates, we can be sure that we haven't broken anything.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/TestReuseActions.yml [hunk 1]
```diff
@@ -426,7 +426,33 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        job: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23]
+        job:
```
Comment (riodeuno) [inline .github/workflows/TestReuseActions.yml:429]: @sharat87 @nidhi-nair Will this be a problem?

Comment (riodeuno) [reply .github/workflows/TestReuseActions.yml:429]: My editor did this automatically.

Comment (sharat87) [reply .github/workflows/TestReuseActions.yml:429]: There's a _tiny_ chance that it might. Sorry can we keep it as is please?

Comment (riodeuno) [reply .github/workflows/TestReuseActions.yml]: Done. Please review again.

**Review by sharat87 — APPROVED** [review]

**Review by mohanarpit — APPROVED** [review]

**Pull request merged**